examples/abitest/module/rust/src/lib.rs

```diff
@@ -108,7 +115,180 @@ impl OakABITestServiceNode for FrontendNode {
     }
 }
 
+// Helper for status conversion
+fn status_convert<T>(result: Result<T, OakStatus>) -> std::io::Result<T> {
+    match result {
+        Ok(t) => Ok(t),
+        Err(status) => Err(std::io::Error::new(
+            std::io::ErrorKind::Other,
+            format!("failure {:?}", status),
+        )),
+    }
+}
+
 impl FrontendNode {
+    fn test_channel_find(&mut self) -> std::io::Result<()> {
+        // Idempotent result.
+        expect_eq!(self.grpc_in, oak::channel_find("gRPC_input"));
+        // Whitespace is significant.
+        expect_eq!(0, oak::channel_find(" gRPC_input"));
+        expect_eq!(0, oak::channel_find(" gRPC_input "));
+        expect_eq!(0, oak::channel_find("bogus"));
+        expect_eq!(0, oak::channel_find(""));
+        Ok(())
+    }
+
+    fn test_channel_create(&mut self) -> std::io::Result<()> {
+        let mut handles = Vec::<(oak::Handle, oak::Handle)>::new();
+        for _i in 0..500 {
+            handles.push(oak::channel_create().unwrap());
```

Comment:
This will panic if channel_create fails. Is that what we want? (vs having an expect OK after each call)

Reply:
Yeah, that should get caught more nicely - fixed.